### PR TITLE
Fix private endpoints

### PR DESCRIPTION
**Pull request by coderGo93**
## Description
Added another parameter called `ID` for body request  of `InterfaceEndpointConnection`

Link to any related issue(s): 

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by themantissa — APPROVED** [review]
LGTM, just wait for review from 2 other reviewers. Thanks!

**Review by gssbzn — APPROVED** [review]
LGTM

File: mongodbatlas/private_endpoints.go
Comment (leofigy) [inline mongodbatlas/private_endpoints.go:47]: looks good just curious is the id really optional?

**Review by leofigy — APPROVED** [review]

File: mongodbatlas/private_endpoints.go
Comment (coderGo93) [reply mongodbatlas/private_endpoints.go:47]: It's required to pass in body request.

**Pull request merged**
